service/grails-app/migrations/module-tenant-changelog.groovy

```diff
@@ -6,4 +6,5 @@ databaseChangeLog = {
   include file: 'docs.groovy'
   include file: 'update-mod-license-1-5-0.groovy'
   include file: 'update-mod-license-1-6.groovy'
+  include file: 'update-mod-license-1-6-1.groovy'
```

Comment:
Can you just append your changes to 'update-mod-license-1-6.groovy' please?
I changed this recently to keep the number of files down. 1.6 is designed as an umbrella for all changes in 1.6.x

Reply:
Looking over the changes needed, the only change here would be to change lic_description from CLOB to TEXT,  which doesn't actually address the jira issue.   Unless there is a desire to always use TEXT instead of CLOB for consistency, no changes to mod-licenses are needed.

Reply:
Okay, in that case there are no changes needed for ERM-218 in mod-licenses and this PR can be removed.   I'll make those changes for the sa_description and res_description fields in the domain classes in mod-agreements and update the migration files.  Should be ready soon.